app/models/pet.rb

```diff
@@ -32,9 +34,12 @@ class Pet < ApplicationRecord
 
   validates :name, presence: true
   validates :birth_date, presence: true
-  validates :size, presence: true
   validates :breed, presence: true
   validates :sex, presence: true
+  validates :weight_from, presence: true
+  validates :weight_to, presence: true
+  validates :weight_unit, presence: true
```

Comment:
Let's add the inclusion to make sure it only is the two values we expect - https://guides.rubyonrails.org/active_record_validations.html#inclusion

Reply:
Good call. Done.